.github/workflows/build_workflow.yml

```diff
@@ -69,8 +69,8 @@ jobs:
       run: echo -e "user $PAUSE_USER\npassword $PAUSE_PASSWORD\n" >> ~/.pause
     - name: setup git
       run: |
-        git config --global user.email "circleci@binary.com"
-        git config --global user.name "circleci"
+        git config --global user.email "github-actions@binary.com"
```

Comment:
Just curious is this github-actions email exist?

Reply:
no. Its just a fake user. The circleci user also seems to be a fake user just to denote the change was made by a CI system